### DISCUSSION TITLE
Supported VPN

### DISCUSSION THREAD

**Discussion by g33kroid** [Security-Onion-Solutions/securityonion#2142] [Unsupported Versions] [answered]
I have tried multiple VPN to connect Remote Sensor to Master (Wire Guard , ZeroTier and OpenVPN) but is there a compatible VPN or proper documentation to configure it

**Comment by TOoSmOotH** [comment, marked as answer]
There is an issue in for this:
https://github.com/Security-Onion-Solutions/securityonion/issues/1139

**Reply by g33kroid** [reply]
I saw it but I still didn't find proper answer thats why I was asking if OpenVPN has issues and P2P VPN (Wire Guard or ZeroTier) maybe there is something else supported

**Reply by TOoSmOotH** [reply]
Not at this time. The issue is really dealing with all of the firewall and IP related stuff so the underlying VPN won't really matter.
